python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -171,31 +49,26 @@ class CuGraphRelGraphConv(nn.Module):
         Default: ``None``.
     bias : bool, optional
         True if bias is added. Default: ``True``.
-    activation : callable, optional
-        Activation function. Default: ``None``.
     self_loop : bool, optional
         True to include self loop message. Default: ``True``.
     dropout : float, optional
         Dropout rate. Default: ``0.0``.
-    layer_norm : bool, optional
```

Comment:
Is this a breaking change?

Reply:
Yes, same as the "activation" argument above.